app/assets/stylesheets/govuk-component/_related-items.scss

```diff
@@ -0,0 +1,32 @@
+.govuk-related-items {
+  border-top: 10px solid $mainstream-brand;
+  padding-top: 5px;
+
+  h2 {
+    @include core-24;
+    font-weight: 600;
```

Comment:
Is there a reason for using 600 here? 700 maps to bold and there's a bold mixin for 24px text which would allow these two lines to be combined.

`@include bold-24`  from the [front end toolkit's _typography.scss](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_typography.scss#L114).